src/NServiceBus.Metrics.PerformanceCounters.Tests/IntegrationTests.cs

```diff
@@ -42,11 +42,13 @@ await endpoint.Stop()
             .ConfigureAwait(false);
 
         var criticalTimePerfCounter = new PerformanceCounter("NServiceBus", PerformanceCountersFeature.CriticalTimeCounterName, endpointName, true);
+        //var processingTimePerfCounter = new PerformanceCounter("NServiceBus", PerformanceCountersFeature.ProcessingTimeCounterName, endpointName, true);
```

Comment:
Yes and no. As @danielmarbach mentions, the tests were pretty useless, so I wondered what to do with those. And after wondering, I indeed forgot about them :-)